recipe/meta.yaml

```diff
@@ -21,6 +21,7 @@ requirements:
     - pip
     - python
   run:
+    - ipython
```

Comment:
It seems ipython is a new dependency (not currently listed in setup.py) is this correct @hanjinliu ?

Reply:
No, ipython is an optional one (macrokit only provides some magic commands)